src/main/flight/mixer.c

```diff
@@ -192,13 +192,50 @@ static inline void mixerApplyInputLimit(int index, float value)
     }
 }
 
+static float mixerGetPassthroughInput(int index, float value)
+{
+    float rc = 0;
+    switch (index) {
+    case MIXER_IN_STABILIZED_ROLL:
+        rc = getRcDeflection(ROLL);
+        break;
+    case MIXER_IN_STABILIZED_PITCH:
+        rc = getRcDeflection(PITCH);
+        break;
+    case MIXER_IN_STABILIZED_YAW:
+        // Normally, yaw command is reversed in setpoint.c (unlike other axes).
+        // As we passthrough RC commands we want to keep the same reversal.
+        rc = -getRcDeflection(YAW);
+        break;
+    case MIXER_IN_STABILIZED_COLLECTIVE:
+        rc = getRcDeflection(COLLECTIVE);
+        break;
+    default:
+        return value;
+    }
+
+    // Scale rc by 120% for easier observing endpoints.
+    rc *= 1.2f;
+
+    if (rc > 0) {
+        value = scaleRangef(rc, 0, 1.0f, 0, mixerInputs(index)->max / 1000.0f);
```

Comment:
Please don't reuse function arguments as variables. IT's a bad habit.

Reply:
Resolved